test/session_spec.lua

```diff
@@ -18,7 +19,7 @@ end
 local function test_session(description, session_factory, session_destroy)
   local get_api_info = function (session)
     local ok, res = session:request('vim_get_api_info')
-    return ok, unpack(res)
+    return ok, _compat.unpack(res)
```

Comment:
this shouldn't be necessary because `unpack` (from `_compat.lua`) goes into the global namespace. That's intentional, otherwise a compat module that requires every call site to be updated won't be very useful..